[settings]
known_third_party = exceptions,fastapi,pydantic,sqlalchemy
